### ISSUE TITLE
Create a property with its attribute is dict

### ISSUE THREAD

**Issue by zhangtianye** [sidorares/node-dbus#91]
How can I create a property in dict ?
In your server2.js, I change TestProperty with signature 'a{sv}', is it right ?
Can you give me a example to create a property in dict ?
Here is my code:
var dbus = require('dbus-native');
var bus = dbus.sessionBus();
var name = 'webrt.service';
bus.requestName(name, 0);

var exampleIface = {
    name: 'webrt.service',	//
    methods: {	//method
        doStuff: ['s', 's'],	
        timesTwo: ['d', 'd'],
        respondWithDouble: ['s', 'd']
    },
    signals: {
        testsignal: [ 'us', 'name1', 'name2' ]
    },
    properties: {
       TestProperty2: 'a{sv}'
    }
};

var example = {
    respondWithDouble: function(s) {
        console.log('Received "' + s + "'");
        return 3.14159;
    },
    timesTwo: function(d) {
	console.log(d);
        return d*2;	
    },
    doStuff: function(s) {
        return 'Received "' + s + '" - this is a reply'; 
    },
    TestProperty2: [],
    emit: function(name, param1, param2) {
        console.log('signal emit', name, param1, param2);
    }
};
bus.exportInterface(example, '/webrt/service', exampleIface);
//	bus.exportInterface(example, '/com/github/sidorares/2', exampleIface);

//setInterval( function() {
//    example.emit('testsignal', Date.now(), 'param2');
//}, 1000);

//});
//

**Comment by sidorares** [comment]
Yes — `'a{sv}'` is the right signature, and it works. Verified against `dbus-daemon` on 0.11:

```js
properties: { Config: 'a{sv}' }

const impl = { Config: { mode: 'auto', retries: 3 } };   // just a plain object
```

```
Config property -> {"mode":"auto","retries":3}
<property name="Config" type="a{sv}" access="readwrite"/>
```

Two things had to land for that to be true, which is why it did not work in 2015:

- **#335 (0.11)** — a plain JS object is now accepted anywhere a dict is expected, with the value signatures inferred (`'auto'` → `s`, `3` → `i`). Before this you had to build `[['mode', ['s', 'auto']], …]` by hand.
- **#330 (0.9)** — property access is declarable and enforced, and `PropertiesChanged` is emitted on write.

Use `Variant` when the inferred type is not the one you want:

```js
const { Variant } = require('dbus-native');
impl.Config = { retries: new Variant('u', 3) };   // uint32, not int32
```

Reading it back still gives the classic array-of-pairs shape until 2.0 — `toPlain()` converts it today and becomes the identity afterwards. That remaining half is #3.

Docs: [writing a dict as a plain object](https://github.com/sidorares/dbus-native/blob/master/docs/api.md#writing-a-dict-as-a-plain-object).
